### DISCUSSION TITLE
In Settings, in the "Window Throw" tab, for the Trackpad option, what does "Rest 3 Touches, Remove All Except 1" mean?

### DISCUSSION THREAD

**Discussion by deslee** [rxhanson/RectanglePro-Community#578] [Q&A] [answered]
I understand the "Window Throw" feature, but I have no idea what the options for "Trackpad" mean or do. Maybe an info tooltip would be helpful, because I couldn't find anything about it by searching the internet or docs either

**Comment by rxhanson** [comment, marked as answer]
On this page there's a video about the Window Throw where the trackpad gestures are explained, it's at about 40s in: https://rectangleapp.com/comparison

Here's the YouTube link: https://www.youtube.com/watch?v=Vo09H-ln7BQ

At some point, I plan to redo parts of the UI to add more and better explanations.

**Reply by deslee** [reply]
Thanks, I figured it out: "Rest 3 touches, remove all except 1" = "place 3 finders on touchpad, then immediately swipe with one finger. That makes sense when reading it in the context of a trackpad _gesture_ which I wasn't
